### ISSUE TITLE
[Update] Request for update to EFO_0920061 perimenopausal

### ISSUE THREAD

**Issue by paola-scibite** [EBISPOT/efo#2678]
### Term to be updated

EFO_0920061 perimenopausal

### Change(s) to EFO term(s)

- [ ] Typo (label, definition, synonym etc.)
- [ ] Incorrect annotation (xref, synonym, etc.)
- [ ] New annotation (xref,synonym, etc.)
- [ ] Update label
- [ ] Update parent term(s)
- [ ] Update child term(s)
- [x] Other

### Comment

EFO_0920061 perimenopausal is currently defined as:
"The transitional phase before menopause, characterized by declining and fluctuating estrogen levels and associated symptoms such as irregular menstrual periods, hot flashes, night sweats, and mood changes. This phase typically begins several years before menopause and ends with the last menstrual period."

However, perimenopause covers times _around_ the last menstrual period, not just before; it doesn't end with it. The definition needs revising please. 

Menopause terminology is confusing, applied inconsistently, and strictly speaking can only be applied a year after what turns out to be the last menstrual period, so there's plenty of cause for confusion and I'm not criticizing! I'll attach a screenshot from [a Nature review](https://www.nature.com/articles/nrdp20154) that I find correct and that you could use as a reference, but it seems to be behind a paywall. (FMP in the figure = final menstrual period.) I can find a better one but not today I'm afraid.  

Thanks,
Paola (Roncaglia)

<img width="869" height="488" alt="Image" src="https://github.com/user-attachments/assets/0dc06fb5-4973-4521-a4c3-688a8abf2d81" />

**Comment by paola-scibite** [comment]
HPO has [HP:0040406 Perimenopausal](https://hpo.jax.org/browse/term/HP:0040406), defined as:

"Perimenopause, or the menopausal transition, encompasses that period of time during which physiologic changes mark progression toward a woman's final menstrual period. This phase begins with the onset of menstrual irregularities and continues until a woman reaches menopause, or one year after amenorrhea has occurred. Perimenopause can last for a variable amount of time, the median of which is four years."

Not sure if there were reasons to create an EFO term rather than importing the HPO one. 
But the HPO definition well reflects the "imprecise" nature of the time limits. 
If the EFO term is kept, please consider removing "length" from "...beginning with changes in menstrual cycle length..."; "changes in menstrual cycle" is a better, more comprehensive summary. 
Also, `perimenopause` and `menopausal transition` could be synonyms, respectively exact and narrow.

Thank you!

**Comment by jamesamcl** [comment]
Triage summary for EFO_0920061 `perimenopausal` (opened by @paola-scibite; assignee @ar-ibrahim).

**Status: not yet applied.** `master` and the live EFO 3.93.0 still carry the original definition ("…ends with the last menstrual period"). A Copilot PR, #2679, is linked to this issue and open since 2026-04-21, but it (a) still uses the wording "menstrual cycle length", (b) does not add the `menopausal transition` synonym, (c) has no review or CI run, and (d) now conflicts with `master` (only in the end-of-file OWL API comment, but it cannot merge as-is; the branch is ~130 commits behind). It would be simplest to redo the edit on a fresh branch and close #2679 as superseded.

**The request is correct.** STRAW+10 (Harlow et al. 2012, PMID:22343510) states that perimenopause "means the time around menopause and begins at Stage –2 and ends 12 months after the FMP", while the menopausal transition (Stages –2/–1) ends at the FMP. HP:0040406 gives the same end point. So `menopausal transition` is strictly narrower than `perimenopause`, and `hasNarrowSynonym` is the right typing, as proposed.

**Import vs keep:** keeping the EFO-native term looks right. HP:0040406 sits under HP:0032443 "Past medical history" (a clinical-history modifier branch, not a life stage), whereas EFO's `premenopausal` (EFO_0920060) and `postmenopausal` (EFO_0002721) are EFO-native children of `adult` (EFO_0001272) and #2653 asked for a sibling. HP:0040406 is already an xref on the term.

**Proposed edit** (`src/ontology/efo-edit.owl`, EFO_0920061 — parent, xrefs, creator unchanged):

1. Replace the definition in both places (the `obo:IAO_0000115` literal and the `owl:annotatedTarget` of its reified axiom):
   > The phase around menopause that spans the menopausal transition and the first 12 months after the final menstrual period. It begins with the onset of menstrual cycle irregularity and ends 12 months after the final menstrual period, when menopause is confirmed retrospectively. It is characterized by fluctuating and declining estrogen levels and may include symptoms such as irregular menstrual periods, hot flashes, night sweats, sleep disturbance, and mood changes.

   Add definition xrefs PMID:22343510 (STRAW+10) and PMID:27188659 (Davis et al. 2015, Nat Rev Dis Primers — the review cited above). Curator decision: whether to keep the three existing PMIDs (41807917, 41829913, 41836925), which are 2026 usage papers rather than definitional sources.
2. `perimenopause` is already an exact synonym — nothing to do.
3. Add `menopausal transition` as `oboInOwl:hasNarrowSynonym`, with a source xref (PMID:22343510; NCIt:C15692 also carries "Menopause Transitional Phase"). Optionally add `menopause transitional phase` (NCIt:C15692) as a further narrow synonym.
4. Optionally add `rdfs:seeAlso https://github.com/EBISPOT/efo/issues/2678` alongside the existing seeAlso for #2653.

Then `om make normalize_src` and `om reason` as usual.

*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
